src/administration/from_scratch.md

```diff
@@ -221,6 +221,9 @@ Now open your Lemmy domain in the browser, and it should show you a configuratio
 
 Compile and install lemmy_server changes. This compile can be done by a normal unprivledged user (using the same Linux account you used for rustup and first install of Lemmy).
 
+IMPORTANT NOTE: Ubuntu 20.04 comes with PostgreSQL 13.x, and Lemmy 0.18.3 and later requires newer versions. It may be best to consult Lemmy support forums on the topic of ugprades using From Scratch if you need help confirming PostgreSQL compatibility.
```

Comment:
Postgres 15 is required since Lemmy version 0.17

https://join-lemmy.org/news/2023-01-31_-_Lemmy_Release_v0.17.0